tests/dt/test-countna.py

```diff
@@ -71,6 +71,12 @@ def test_dt_count_na1(src):
 
 def test_dt_count_na2():
     DT = dt.Frame(G=[1,1,1,2,2,2], V=[None, None, None, None, 3, 5])
-    EXP = dt.Frame(G=[1,2], V1=[3,1], V2=[3,0])
+    EXP = dt.Frame(G=[1,2], V1=[3,1], V2=[1,0])
```

Comment:
Hm, I wonder if that was a bug?

Reply:
yea, it was. I didnt think of it until i compared results. The other surprising part for me is the fact that nested groupby works here with no drama, unlike sum/prod/min/max ... any ideas?

Reply:
Then may be open an issue illustrating the bug, so that we knew it is fixed by this PR. Do you know what part of the C++ code had the bug?

As for the aggregations errors, I actually found the reason why it errors, but had no time to fix that. I will work on that soon.

Reply:
no idea on what part of the c++ code caused the bug. Do i link it to the existing [issue](#3417 )?

Reply:
Yes, you can. It is still a bug in `countna()`, just add an example.